src/ralph/defaults.py

```diff
@@ -99,7 +99,6 @@ def config(key, default_value):
 CONFIG_FILE = APP_DIR / "config.yml"
 CONFIG = load_config(CONFIG_FILE)
 ENVVAR_PREFIX = "RALPH"
-DEFAULT_GELF_PARSER_CHUNCK_SIZE = config("RALPH_DEFAULT_GELF_PARSER_CHUNCK_SIZE", 5000)
 DEFAULT_BACKEND_CHUNCK_SIZE = config("RALPH_DEFAULT_BACKEND_CHUNCK_SIZE", 500)
```

Comment:
@quitterie-lcs has already seen this one?

```suggestion
DEFAULT_BACKEND_CHUNK_SIZE = config("RALPH_DEFAULT_BACKEND_CHUNK_SIZE", 500) 
```

Reply:
It's from here: https://github.com/openfun/ralph/pull/74#discussion_r597672728

Reply:
So will you fix it here? or in another PR?

Reply:
Ah! It's done!

Reply:
But in the next commit. :sweat_smile: